<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="menu" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QPushButton" name="add_pasient_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingresar
Paciente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_sala_info_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Info de
Sala Solicitada </string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_id_info_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Info de 
Paciente segun
ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="aditional_inf_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Aditional Info</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
